core/views.py

```diff
@@ -53,41 +53,50 @@ def _get_friends(pk):
                                             Q(status='y'))
     friends = []
     for f in friendships:
-        friends.append((f.sent_by.username, f.sent_by.online) if not f.sent_by.username == self_username else
-                       (f.sent_to.username, f.sent_to.online))
+        if not f.sent_by.username == self_username:
+            friends.append((
+                f.sent_by.username,
+                f.sent_by.online,
+                PongUser.objects.get(username=f.sent_by.username).get_wins().count(),
+                PongUser.objects.get(username=f.sent_by.username).get_losses().count(),
+                PongUser.objects.get(username=f.sent_by.username).get_winrate(),
+            ))
+        else:
+            friends.append((
+                f.sent_to.username,
+                f.sent_to.online,
+                PongUser.objects.get(username=f.sent_to.username).get_wins().count(),
+                PongUser.objects.get(username=f.sent_to.username).get_losses().count(),
+                PongUser.objects.get(username=f.sent_to.username).get_winrate(),
+            ))
```

Comment:
`friends.append(...) if not f.sent_by_username == self_username else friends.append(...)`